packages/@aws-cdk/aws-redshift/package.json

```diff
@@ -82,20 +82,52 @@
     "@aws-cdk/assert": "0.0.0",
     "cdk-build-tools": "0.0.0",
     "cfn2ts": "0.0.0",
+    "jest": "^24.9.0",
     "pkglint": "0.0.0"
   },
   "dependencies": {
+    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
+    "@aws-cdk/aws-secretsmanager": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.0.2"
   },
   "homepage": "https://github.com/aws/aws-cdk",
   "peerDependencies": {
+    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
+    "@aws-cdk/aws-secretsmanager": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.0.2"
   },
   "engines": {
     "node": ">= 10.12.0"
   },
+  "awslint": {
+    "exclude": [
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupFamily.REDSHIFT_1_0",
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupFamily.DUMMY",
+      "docs-public-apis:@aws-cdk/aws-redshift.ClusterType.MULTI_NODE",
+      "docs-public-apis:@aws-cdk/aws-redshift.ClusterType.SINGLE_NODE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC1_8XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC1_LARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC2_8XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC2_LARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DS2_8XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DS2_XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.RA3_16XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupParameters.parameterName",
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupParameters.parameterValue",
+      "props-struct-name:@aws-cdk/aws-redshift.IClusterParameterGroup",
+      "props-physical-name:@aws-cdk/aws-redshift.ClusterParameterGroupProps",
+      "props-physical-name:@aws-cdk/aws-redshift.DatabaseSecretProps",
+      "props-physical-name:@aws-cdk/aws-redshift.ClusterProps"
```

Comment:
If you'll follow my naming guidelines, those exclusions will not be needed :).

Reply:
Dont really know what you mean...sorry, I need more to go on.
`package.json`s all over the place are riddled with awslint excludes like that (i.e. `props-physical-name:...`) .

Reply:
For `"props-struct-name:@aws-cdk/aws-redshift.IClusterParameterGroup"`, I believe this exclusions is not needed (and if it is, please let me know what error if fails with, because I think it should work!).

For `"props-physical-name:@aws-cdk/aws-redshift.ClusterParameterGroupProps"`, that's required, as apparently `ClusterParameterGroup`s don't allow setting a physical name. Same for `DatabaseSecretProps`.

But I think for `ClusterProps`, if you rename `clusterIdentifier` to `clusterName`, that exclusion can be removed.

Reply:
OK. Cool. Sounds like a plan :)